src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const Todo = ({ todos, deleteTodo }) => {
+  const todoList = todos.length ? (
+    todos.map(todo => {
+      return (
+        <div className="item" key={todo.id}>
+          <span onClick={()=>{deleteTodo(todo.id)}}>{todo.task}</span>
+          <button onClick={()=>{deleteTodo(todo.id)}}>CLEAR COMPLETED</button>
+        </div>
```

Comment:
since you have the function in two places, you could also give the parent tag that function instead.